lib/auth0_ex/parser.ex

```diff
@@ -42,5 +42,5 @@ defmodule Auth0Ex.Parser do
     do:
       headers
       |> Map.new()
-      |> Map.get("Content-Type") == "application/json"
+      |> Map.get("Content-Type") =~ "application/json"
```

Comment:
this has been taken care by another change. would you mind removing that? We can still get the organization change in